lib/howitzer/meta/actions.rb

```diff
@@ -0,0 +1,30 @@
+module Howitzer
+  module Meta
+    # Module with utility actions for elements
+    module Actions
+      # Highlights element with red border on the page
+      # @param *args [Array] arguments for elements described with lambda locators
```

Comment:
Here and at other places: documentation is wrong. In fact, it can be argumens for elements described with lambda **+ inline options for element/s as a hash**